src/Mjolnir.ts

```diff
@@ -836,6 +838,36 @@ export class Mjolnir {
         await this.printBanlistChanges(changes, banList, true);
     }
 
+    private async handleConsequence(protection: Protection, roomId: string, eventId: string, sender: string, consequence: Consequence) {
+        switch (consequence.type) {
+            case ConsequenceType.alert:
```

Comment:
Shouldn't we also alert here?
Otherwise, can you add a comment explaining that we're expecting the protection to have already raised the alarm?

Reply:
"alert" is basically a no-op. each consequence announces itself, but "alert" is just the announcement

Reply:
Could you add a comment explaining that?